user_docs/guides/logging_out_and_server_shutdown.md

```diff
@@ -0,0 +1,29 @@
+[![OpenSARlab Header](../assets/OSL_user_guide_header.png)](../OpenSARlab_user_guide.md)
+
+# Logging Out of OpenSARlab and Shutting Down the Server
+
+When you are ready to stop working in OpenSARlab, it is best practice to shut down your server and also logout.
+
+## Why Shut Down the Server?
+Logging out will not shut down the server. If the server is inactive for an hour, it will shut down automatically, but users should not rely upon this feature. If there are any running processes in your account, they will keep the server alive, and it is not always apparent that there is a process running. If you login for the first time in a day or two and notice that a Jupyter Notebook is still running, your server ran that whole time. 
+
+Do your part to reduce resource use and ease the burden on the environment by shutting down your server when you are finished working for the day. 
+
+## Why Doesn't OpenSARlab Shut Down the Server Automatically Upon Logout?
+In some instances you may need to leave your server running, which is perfectly acceptable. Perhaps you have a notebook performing a very time intensive analysis and wish to let it run overnight.
```

Comment:
Why doesn't the server shutdown upon logout? Because that's what it's suppose to do. We actually can make it shut down on logout but I think that would not be good for the long running processes.

Instead of a question, I think it would be better if this was a **bolded statement** since it really follows from the previous question.

Reply:
I made that update. What do you think about the description of when the server shuts down automatically. Based on the docs you pointed out it is incorrect, but it is the behavior I have seen. Maybe we should do some experimenting and nail down what is causing these phantom instances to stay alive.